test/channels/retro_channel_test.exs

```diff
@@ -46,13 +46,6 @@ defmodule RemoteRetro.RetroChannelTest do
 
   describe "pushing `proceed_to_next_stage` with a stage of 'action-item-distribution'" do
     setup [:join_the_retro_channel]
-
```

Comment:
just pulled this branch to do the rebase, and @Zanadar, *now* i get why you're talking about this test needing inserting test records. the test was failing because there was no-one in the `to` field of the email, which in turn was caused by there being no one in the test database. my bad.